YoutubeDownloader/Services/QueryService.cs

```diff
@@ -63,6 +69,15 @@ public async Task<ExecutedQuery> ExecuteQueryAsync(Query query)
                 return new ExecutedQuery(query, title, playlist.Videos);
             }
 
+            // Channel
+            if (query.Type == QueryType.Channel)
+            {
+                var playlist = await _youtubeClient.GetChannelUploadsAsync(query.Value);
+                var title = $"Total {playlist.Count} videos";
```

Comment:
Can you change the title to "Channel uploads" (without count) for more consistency?